procfile/README.md

```diff
@@ -0,0 +1,52 @@
+# Procfile
+
+This is a Go library for parsing the Procfile format.
+
+## Formats
+
+### Standard
+
+The standard Procfile format is what you're probably most familiar with, which maps a process name to the command to run. An example of a standard Procfile might look like:
+
+```yaml
+web: ./bin/web
+worker: ./bin/worker
+```
+
+The standard Procfile format is specified in https://devcenter.heroku.com/articles/procfile.
+
+### Extended
+
+The extended Procfile format is Empire specific and implements a subset of the attributes defined in the [docker-compose.yml](https://docs.docker.com/compose/yml/) format. The extended Procfile format gives you more control, and allows you to configure things like health checks for the individual processes. An example of an extended Procfile might look like:
+
+```yaml
+web:
+  command: ./bin/web
+  health_checks:
```

Comment:
Note that I didn't nest this under a `load_balancer` key, which is talked about in https://github.com/remind101/empire/issues/491. I did this mostly to avoid coupling us to the idea of a load balancer, since it's kind of implementation specific (maybe we'll support some network overlay in the future? Who knows). This also makes it possible to add health checks to non-web processes.